lib/main.js

```diff
@@ -60,18 +60,24 @@ export default {
       lint: (activeEditor) => {
         const command = atom.config.get("linter-clang.execPath");
         const file = activeEditor.getPath();
+        const ext = (file.match(/\.([^.]+)$/) || [])[1];
         const args = ["-fsyntax-only",
           "-fno-caret-diagnostics",
           "-fno-diagnostics-fixit-info",
           "-fdiagnostics-print-source-range-info",
           "-fexceptions"];
 
+        let filterRegex = null;
         const grammar = activeEditor.getGrammar().name;
 
         if(/^C\+\+/.test(grammar)) {
           //const language = "c++";
           args.push("-xc++");
           args.push(...atom.config.get("linter-clang.clangDefaultCppFlags").split(/\s+/));
+          
```

Comment:
You have blank whitespace on this line.
![image](https://cloud.githubusercontent.com/assets/427137/11513944/d9171208-982a-11e5-957b-249fa7c0ec36.png)